src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -234,4 +248,42 @@ void testIsPaperCaseAttending() {
 
         assertThat(result).isFalse();
     }
+
+
+    private static Stream<Arguments> buildHearingChannelRequests() {
+        return Stream.of(
+            Arguments.of(FACE_TO_FACE),
+            Arguments.of(VIDEO),
+            Arguments.of(PAPER),
+            Arguments.of(TELEPHONE)
+        );
+    }
+
```

Comment:
Please add tests for remaining scenarios as mentioned earlier. See https://sonarcloud.io/component_measures?metric=new_coverage&selected=SSCSHA%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fsscs%2Fhelper%2Fmapping%2FHearingsChannelMapping.java&view=list&pullRequest=427&id=SSCSHA for code coverage.

Reply:
https://sonarcloud.io/component_measures?metric=new_coverage&selected=SSCSHA%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fsscs&view=list&pullRequest=427&id=SSCSHA

Reply:
Updated the test cases.

Reply:
Coverage is 100% now